ecommerce/api.py

```diff
@@ -137,10 +136,12 @@ def create_unfulfilled_order(course_id, user):
     )
     Line.objects.create(
         order=order,
-        course_key=course_run.edx_course_key,
+        course_key=course_key,
         description='Seat for {}'.format(course_run.title),
         price=price,
     )
+    if coupon is not None:
+        RedeemedCoupon.objects.create(order=order, coupon=coupon)
```

Comment:
`order` is created in this function so there shouldn't be a row in the database already